server/src/main/java/io/crate/expression/symbol/SymbolType.java

```diff
@@ -44,6 +44,7 @@
     LITERAL(Literal::new),
     INPUT_COLUMN(InputColumn::new),
     DYNAMIC_REFERENCE(DynamicReference::new),
+    VOID_REFERENCE(VoidReference::new),
```

Comment:
This must be added last, otherwise is causes a streaming change because the oridnal is used to stream the symbol-type, and the value would shift by one for each of the following definitions.